tox.ini

```diff
@@ -46,6 +46,40 @@ commands =
     python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/nvtabular.git
     horovodrun -np 2 sh examples/usecases/multi-gpu/hvd_wrapper.sh python -m pytest -m horovod -rxs tests/unit
 
+[testenv:py38-nvtabular-cpu]
+passenv=GIT_COMMIT
+allowlist_externals = git
+deps =
+    -rrequirements/base.txt
+    -rrequirements/dev.txt
+commands =
+    ; the GIT_COMMIT env is the current commit of the models repo
+    git clone --depth 1 --branch {posargs:main} https://github.com/NVIDIA-Merlin/NVTabular.git nvtabular-{env:GIT_COMMIT}
```

Comment:
I think we'd want to clone/install the `GIT_COMMIT` version of the models repo here, no? We are running the NVT tests, but want _this_ commit of models. (same thing for the gpu tests below)

Reply:
Isn't the `GIT_COMMIT` version of Models already cloned by Github Actions automatically to the current directory? And we install this version of Models in line 63 with `pip install .`.

I mostly copy-pasted how it's set up in [Core](https://github.com/NVIDIA-Merlin/core/blob/cd96ca5f7ec6e9d26eba1257a8b1293b6d1bd7b3/tox.ini#L34-L51) but the workflow looks like:
- Line 57: clone NVT tests.
- Line 58-62: Install dependencies. (Just noticed that we don't need line 62. I will remove that).
- Line 63: GHA already cloned the `GIT_COMMIT` version to the current directory, so we just need to run `pip install .` to install Models.
- Line 64: Run NVT tests (which will use the version of Models installed in line 63 with `pip install .`).

What do you think?

Reply:
Ah yes I think I was thrown by the existence of line 62. Yeah this makes sense to me.